lib/network/kmip/codec/ttlv.ts

```diff
@@ -369,7 +369,7 @@ function TTLVCodec() {
 
 
     /* Public Methods Definition */
-
+    // @ts-ignore
```

Comment:
Instead of ts-ignores, if this is a rule we don't want, why not just adding a one liner comment at the top of the file or in the `tsconfig.json` file?